src/systems/ec/implementations/react/packages/file/FileDownload.jsx

```diff
@@ -70,7 +71,7 @@ const FileDownload = ({
                 key={item.title}
               >
                 <div className="ecl-file__translation-info">
-                  <div className="ecl-file__translation-title">
+                  <div className="ecl-file__translation-title" lang={item.lang}>
```

Comment:
`lang` should be used only if `item.lang` is a non-empty string

btw `item.lang` is missing in the props validation